Vagrantfile

```diff
@@ -0,0 +1,62 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV['VAGRANT_DEFAULT_PROVIDER'] ||= "virtualbox"
+DOCKER_PORT=2376
+UNICORN_PORT=3000
+VAGRANTFILE_API_VERSION = "2"
+
+$docker_setup=<<CONF
+cat > /etc/default/docker << 'EOF'
+DOCKER_OPTS="-H 0.0.0.0:#{DOCKER_PORT} -H unix:///var/run/docker.sock"
+EOF
+service docker restart
+CONF
+
+unless Vagrant.has_plugin?("vagrant-cachier")
+  puts "WARNING: vagrant-cachier plugin is not installed! It really speeds this up..."
+  puts "         Install using 'vagrant plugin install vagrant-cachier'"
+end
```

Comment:
I'm not going to worry about this for now - had a quick go with Log4r::Logger and started to go down a rabbit hole. There's something unusual with how vagrant evaluates the Vagrantfile.